JS/contact-form.js

```diff
@@ -0,0 +1,17 @@
+const emailForm = document.querySelector('#e-mail');
+const popupForm = document.querySelector('.formPopup');
+const submit = document.querySelector('#submit');
+
+submit.addEventListener('click', () => {
+  if (/[A-Z]/.test(emailForm.value)) {
+    popupForm.style.display = 'flex';
+  } else {
+    popupForm.style.display = 'none';
+  }
+});
+
+submit.addEventListener('click', (event) => {
+  if (/[A-Z]/.test(emailForm.value)) {
+    event.preventDefault();
+  }
+}, false);
```

Comment:
Nice job with validation 👍. After filling all the input fields with proper data the form submission should be successful but I get an error. Kindly, fix that for normal form submission.

<img width="623" alt="image" src="https://user-images.githubusercontent.com/34578631/175542402-9214ab0a-756c-4fc0-9324-5ff7d3b63537.png">

Reply:
### Thank you @Mre55 for suggesting this form submission changes,
I have really understood what it takes for a form to be submitted on Formspree and I have now created my personal Formspree account to store all successfully submitted data.

***Thank you so much!***  I hope now the issue is solved as suggested 👍